### PR TITLE
feat(portal): Allow creating resources from Resources page

### LINKED ISSUE #4625: Add "Add Resource" button to Resources table

### PR DESCRIPTION
**Pull request by AndrewDryga**
Closes #4625

### CODE REVIEW

File: elixir/mix.lock [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  "bandit": {:hex, :bandit, "1.4.2", "a1475c8dcbffd1f43002797f99487a64c8444753ff2b282b52409e279488e1f5", [:mix], [{:hpax, "~> 0.1.1", [hex: :hpax, repo: "hexpm", optional: false]}, {:plug, "~> 1.14", [hex: :plug, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4 or ~> 1.0", [hex: :telemetry, repo: "hexpm", optional: false]}, {:thousand_island, "~> 1.0", [hex: :thousand_island, repo: "hexpm", optional: false]}, {:websock, "~> 0.5", [hex: :websock, repo: "hexpm", optional: false]}], "hexpm", "3db8bacea631bd926cc62ccad58edfee4252d1b4c5cccbbad9825df2722b884f"},
+  "bandit": {:hex, :bandit, "1.5.0", "3bc864a0da7f013ad3713a7f550c6a6ec0e19b8d8715ec678256a0dc197d5539", [:mix], [{:hpax, "~> 0.1.1", [hex: :hpax, repo: "hexpm", optional: false]}, {:plug, "~> 1.14", [hex: :plug, repo: "hexpm", optional: false]}, {:telemetry, "~> 0.4 or ~> 1.0", [hex: :telemetry, repo: "hexpm", optional: false]}, {:thousand_island, "~> 1.0", [hex: :thousand_island, repo: "hexpm", optional: false]}, {:websock, "~> 0.5", [hex: :websock, repo: "hexpm", optional: false]}], "hexpm", "92d18d9a7228a597e0d4661ef69a874ea82d63ff49c7d801a5c68cb18ebbbd72"},
```
Comment (jamilbk) [inline elixir/mix.lock:4]: No breaking changes this time 👍 

https://hexdocs.pm/bandit/changelog.html#1-5-0-21-apr-2024

File: elixir/apps/web/lib/web/live/resources/components.ex [hunk 2]
```diff
@@ -207,9 +207,33 @@ defmodule Web.Resources.Components do
...
+      type="select"
+      label="Site"
+      name={"#{@form.name}[0][gateway_group_id]"}
+      options={
+        Enum.map(@gateway_groups, fn gateway_group ->
+          {gateway_group.name, gateway_group.id}
+        end)
+      }
+      value={connected_gateway_group_id}
+      placeholder="Site"
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/live/resources/components.ex]: nit:

```suggestion
      placeholder="Select a Site"
```

**Review by jamilbk — APPROVED** [review]
LGTM

File: elixir/mix.lock (hunk 1 above)
Comment (AndrewDryga) [reply elixir/mix.lock:4]: Yeah, I checked it too. We don't use Bandit for anything except liveness probes so not a high risk anyway.

File: elixir/apps/web/lib/web/live/resources/components.ex (hunk 2 above)
Comment (AndrewDryga) [reply elixir/apps/web/lib/web/live/resources/components.ex]: I think placeholder is not shown on `<select>` anyway, I'll just drop it.

**Pull request merged**